modular_skyrat/modules/gun_cargo/code/armament_datums/nanotrasen.dm

```diff
@@ -20,10 +25,20 @@
 	lower_cost = CARGO_CRATE_VALUE * 8
 	upper_cost = CARGO_CRATE_VALUE * 12
 
+/datum/armament_entry/cargo_gun/nanotrasen/shotgun/doublebarrel
+	item_type = /obj/item/gun/ballistic/shotgun/doublebarrel
+	lower_cost = CARGO_CRATE_VALUE * 10
+	upper_cost = CARGO_CRATE_VALUE * 12
+	
 /datum/armament_entry/cargo_gun/nanotrasen/shotgun/combat
 	item_type = /obj/item/gun/ballistic/shotgun/automatic/combat
 	lower_cost = CARGO_CRATE_VALUE * 10
 	upper_cost = CARGO_CRATE_VALUE * 14
+	
+/datum/armament_entry/cargo_gun/nanotrasen/shotgun/dualtube
```

Comment:
ditto
also how does the dualtube differ from the doublebarrel

Reply:
one is a doublebarrel, one is a shotgun with two seperate magazines of 3 each